CHANGELOG: remove release-note-none items

pandaamanda opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

**What type of PR is this?**
/kind cleanup
<!--
Add one of the following kinds:
/kind bug
/kind cleanup
/kind documentation
/kind feature
/kind design

Optionally add one or more of the following kinds if applicable:
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
-->

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

<!--
This section can be blank if this pull request does not require a release note.

When adding links which point to resources within git repositories, like
KEPs or supporting documentation, please reference a specific commit and avoid
linking directly to the master branch. This ensures that links reference a
specific point in time, rather than a document that may change over time.

See here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files

Please use the following format for linking documentation:
- [KEP]: <link>
- [Usage]: <link>
- [Other doc]: <link>
-->
```docs

```

dougsland reviewed (commented):
humm, I am not sure if removing the NONE lines here helps. My guess the "NONE" string is kind of not covered scenario in the script that generates the CHANGELOG. The pull requests URLs are real. I would try to understand how the changelog are generated and try to fix the script (if generated by the script).

pandaamanda commented:
> humm, I am not sure if removing the NONE lines here helps. My [guess](http://track.uc.cn/lct?appid=2725715d66c1&turl=http://m.sm.cn/s?q=guess&from=wh10011&uks=%E5%85%A8%E9%83%A8_guess&cls=0&clt=-1&rf=github.com&kid=1;uc_param_str=dnntnwvepffrgibijbprsvpidsdichei) the "NONE" string is kind of not covered scenario in the script that generates the CHANGELOG. The pull requests URLs are real. I would try to understand how the changelog are generated and try to fix the script (if generated by the script).

pandaamanda commented:
/assign @wilsonehusin

pandaamanda commented:
/retest

pandaamanda committed:
CHANGELOG: remove release-note-none items

CHANGELOG: remove release-note-none items

CHANGELOG: remove release-note-none items

pandaamanda commented:
/retest

wilsonehusin commented on CHANGELOG/CHANGELOG-1.21.md:
```diff
@@ -381,7 +381,6 @@ filename | sha512 hash
 - Migrate log messages in pkg/scheduler/{scheduler.go,factory.go} to structured logging ([#97509](https://github.com/kubernetes/kubernetes/pull/97509), [@aldudko](https://github.com/aldudko)) [SIG Scheduling]
 - Migrate proxy/iptables/proxier.go logs to structured logging ([#97678](https://github.com/kubernetes/kubernetes/pull/97678), [@JornShen](https://github.com/JornShen)) [SIG Network]
 - Migrate some scheduler log messages to structured logging ([#97349](https://github.com/kubernetes/kubernetes/pull/97349), [@aldudko](https://github.com/aldudko)) [SIG Scheduling]
-- NONE ([#97167](https://github.com/kubernetes/kubernetes/pull/97167), [@geegeea](https://github.com/geegeea)) [SIG Node]
```
hmm I think we should leave 1.21 untouched for now since this is pre-release. we also recently just merged a new change to improve this filtering and parsing so hopefully this goes away before the release date.

pandaamanda replied:
OK. i will revert the modification of 1.21 and remove none in 1.9.

wilsonehusin reviewed (changes requested):
Since this covers almost all scenarios where `NONE` appeared in the release note, I'm inclined to accept this PR.

While you're here, can you also remove the ones from 1.9? Seems like there is 4 lines with `NONE`.

(We don't usually modify the CHANGELOG for unmaintained releases, but if we're fixing something of a similar pattern then I think we should do it as much as possible in one PR)

pandaamanda commented:
/retest

wilsonehusin commented:
@pandaamanda no need to retest if you're still making changes, since the whole test suite will need to be re-done anyway before merge.

pandaamanda commented:
@wilsonehusin  done for 1.21 and 1.9.

pandaamanda commented:
/retest

pandaamanda commented:
/retest

wilsonehusin commented on CHANGELOG/CHANGELOG-1.21.md:
```diff
@@ -413,4 +413,4 @@ _Nothing has changed._
 - rsc.io/quote/v3: v3.1.0
-- rsc.io/sampler: v1.3.0
\ No newline at end of file
+- rsc.io/sampler: v1.3.0
```
can we remove this delta as well? 1.21 file should not be here at all

pandaamanda replied:
updated.

wilsonehusin commented:
/lgtm
/approve

/retest

pandaamanda commented:
/retest

pandaamanda commented:
/retest
